Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -39,6 +39,7 @@
 #include "NetworkProcessMessages.h"
 #include "PageLoadState.h"
 #include "RestrictedOpenerType.h"
+#include "ScreenTimeWebsiteDataSupport.h"
```

Comment:
Move this to `#if PLATFORM(COCOA)`, after line 95. Will fix the GTK build failure.